src/game_api/script/lua_vm.cpp

```diff
@@ -661,6 +661,12 @@ end
     lua["waddler_set_entity_meta"] = waddler_set_entity_meta;
     /// Gets the entity type of the item in the provided slot
     lua["waddler_entity_type_in_slot"] = waddler_entity_type_in_slot;
+    /// Draw text on the screen using the built-in renderer. Use in combination with ON.VANILLA_RENDER
+    lua["vanilla_draw_text"] = vanilla_draw_text;
+    /// Measure the provided text using the built-in renderer. Returns the result as if the string was rendered with vanilla_draw_text.
+    lua["vanilla_measure_text"] = vanilla_measure_text;
```

Comment:
The gui version is called _draw_text_size_, might want to stay consistent here too with vanilla_draw_text_size...